docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -111,6 +111,23 @@ Ensure that your firewalls allow inbound DNS traffic on port `53` from the
 instance(s) running Teleport's Windows Desktop Service to the LDAP server
 (Active Directory Domain Controller).
 
+### RDP Connection Failed
```

Comment:
We use sentence case in headings. For some reason, headings in this page use capitalization inconsistently—you could either update just this heading (so we move gradually toward the standard style) or, if you have time, all the headings in the page.